### DISCUSSION TITLE
Import Error: No module named 'PySDM_examples'

### DISCUSSION THREAD

**Discussion by ghost** [open-atmos/PySDM#474] [Q&A] [answered]
The following import line throws missing module error.

```
from PySDM_examples.Shima_et_al_2009.spectrum_plotter import SpectrumPlotter
ModuleNotFoundError: No module named 'PySDM_examples'
```

**Comment by ghost** [comment, marked as answer]
This is really silly of me. PySDM_examples is a seperate package which I had to install using 
`pip install git+https://github.com/atmos-cloud-sim-uj/PySDM-examples.git`

**Comment by slayoo** [comment]
Apologies, this should have been made more clear in the README. We've just updated it (https://github.com/atmos-cloud-sim-uj/PySDM/commit/e9a4a1be389eb04b9a81b6f4ebd1246499f3792f) with the following paragraph:

Running the examples requires the the ``PySDM_examples`` package to be installed (or within ``PYTHONPATH``). Since the examples package includes Jupyter notebooks, the suggested install and launch steps are:
```
git clone https://github.com/atmos-cloud-sim-uj/PySDM-examples.git
cd PySDM-examples
pip install -e .
jupyter-notebook
```
